.github/workflows/integration_test.yaml

```diff
@@ -5,7 +5,11 @@ on:
 
 jobs:
   integration-tests:
-    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
+    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@feat-rockcraft-from-src
```

Comment:
I'm guessing this builds a version of rockcraft that supports what we need given the breaking changes?

Reply:
This is to help the development of some experimental features in rockcraft that are not available in the snap store version of rockcraft, I have a pull request for adding this to `operator-workflows`
https://github.com/canonical/operator-workflows/pull/259